.github/workflows/es-spellcheck.yml

```diff
@@ -11,9 +11,7 @@
 name: Spellcheck CI
 # yamllint disable-line rule:truthy
 on:
-  pull_request:
-    paths:
-      - 'content/es/**/*.md'
+  push:
```

Comment:
[As-is] (on `pull_request:`)
- Spellcheck workflow will be executed for each PR opening towards `dev-es`.
- If spellcheck finds wrong/unknown words, then anyone can notice in each PR's 'Checks' section.
  ![image](https://user-images.githubusercontent.com/46767780/192658174-0c02bfa2-38b2-486e-bc5b-b9885168a64a.png)

---

[To-be] (on `push:`)
- Spellcheck workflow will be executed for each commit push towards `dev-es`, that is, each merging of PR.
- If spellcheck finds wrong/unknown words, then its result can be found manually at [here](https://github.com/cncf/glossary/actions/workflows/es-spellcheck.yml),
  ![image](https://user-images.githubusercontent.com/46767780/192658520-afbdcbfb-f728-46a4-bf40-c94f6453c263.png)
  but to fix that, someone needs to open another PR to fix that spell error, since the original PR that introduced the spell error has been already merged,
  and IMO this is not an appropriate procedure for checking and fixing spell errors in each PR.

@92nqb @ramrodo @electrocucaracha @raelga what do you think about this? 😊

Reply:
Hi @jihoon-seo, you're right the initial behavior was more appropriated, the change made by @92nqb was during the testing phase. The change proposed here will only include a new step in the existing job, without any modification in the triggers.  @92nqb can you change this to reflect that.